doc/user/metrics/service-monitor.md

```diff
@@ -0,0 +1,45 @@
+## Using the ServiceMonitor prometheus-operator CRD
+
+[prometheus-operator][prom-operator] is an operator that creates, configures, and manages Prometheus clusters atop Kubernetes.
+
+`ServiceMonitor` is a [CR][cr] of the prometheus-operator, which discovers the `Endpoints` in `Service` objects and configures Prometheus to monitor those `Pod`s. See the prometheus-operator [documention][service-monitor] to learn more about `ServiceMonitor`s.
```

Comment:
CR==>CustomResource.
I think it's better to use the full name if we haven't defined an acronym before using it in a given doc.
```suggestion
`ServiceMonitor` is a CustomResource of the prometheus-operator, which discovers the `Endpoints` in `Service` objects and configures Prometheus to monitor those `Pod`s. See the prometheus-operator [documention][service-monitor] to learn more about `ServiceMonitor`s.
```
Also just a suggestion but we probably can do without linking to the CustomResource docs here. At this point users should be familiar with CR and CRDs.
Linking to the ServiceMonitor doc is all we need which we've already done.

Reply:
Maybe this is subjective but "`Pod`s" looks a bit odd to me.
Can we change it to either "`Pods`" or just "Pods"(without the highlighting)?
Same for "`Service`s" and "`ServiceMonitor`s"